magefiles/rulesengine/repos/common.go

```diff
@@ -88,6 +88,25 @@ func ExecuteTestAction(rctx *rulesengine.RuleCtx) error {
 
 }
 
+func GetPairedCommitSha(repoForPairing string, rctx *rulesengine.RuleCtx) string {
+	var pullRequests []gh.PullRequest
+
+	url := fmt.Sprintf("https://api.github.com/repos/redhat-appstudio/%s/pulls?per_page=100", repoForPairing)
```

Comment:
should this be also changed to `konflux-ci`? Or not yet. Not idea, just asking.

Reply:
Good catch!  Updated